src/shared/lib/PdbUtils.ts

```diff
@@ -6,6 +6,11 @@ import {
     ALIGNMENT_GAP, ALIGNMENT_MINUS, ALIGNMENT_PLUS, ALIGNMENT_SPACE,
 } from "shared/model/Pdb";
 
+// TODO temporary workaround for pdb structures not well supported by 3Dmol.js
+export const PDB_BLACKLIST = [
```

Comment:
@onursumer maybe better to call this `PDB_IGNORELIST` or `PDB_DENYLIST`. `blacklist/whitelist` is a bit controversial, see e.g. https://github.com/rails/rails/issues/33677

Reply:
lets' do ignore list, that sounds better.

Reply:
@inodb oops I forgot to push my latest changes yesterday, thanks for updating the PR.